src/InteractiveWindow/Editor/InteractiveWindow_UIThread.cs

```diff
@@ -334,6 +334,12 @@ private void PendSubmissions(IEnumerable<PendingSubmission> inputs)
 
             public void AddInput(string command)
             {
+                // If the language buffer is readonly then input can not be added. Return immediately.
+                if (_window._currentLanguageBuffer != null && _window._currentLanguageBuffer.IsReadOnly(0))
```

Comment:
Can you please add a comment about when this occurs?  Something like the description for this PR.